add-on/panels/js/conversationAppStore.js

```diff
@@ -42,8 +42,13 @@ loop.store.ConversationAppStore = (function() {
 
     // Start listening for specific events, coming from the window object.
     this._eventHandlers = {};
-    ["unload", "LoopHangupNow", "socialFrameAttached", "socialFrameDetached", "ToggleBrowserSharing"]
-      .forEach(function(eventName) {
+    [
+      "unload",
+      "LoopHangupNow",
+      "socialFrameAttached",
+      "socialFrameDetached",
+      "ToggleBrowserSharing"
+    ].forEach(function(eventName) {
```

Comment:
yeah, it remained cause my first approach was listen to activeRoom movements, so I did some changes on this file. this one, I forgot.